e3tools/local-development-portal
# Add sync climate contribution from documents

## How to test?

Run  `python manage.py 8_syncclimatecontribution` and check that your investments have been updated
